WcaOnRails/app/mailers/wca_monthly_digest_mailer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class WcaMonthlyDigestMailer < ApplicationMailer
+  include MailersHelper
+
+  def send_weat_digest_content
+    mail(
+      to: ["board@worldcubeassociation.org", "assistants@worldcubeassociation.org"],
+      reply_to: ["board@worldcubeassociation.org", "assistants@worldcubeassociation.org"],
```

Comment:
Does the Board want these emails? Shouldn't the purpose of this digest be to take work off them and let WEAT do the work instead?

Reply:
You are right, it's not needed. I've removed it.